### PR TITLE
Add run command wizard and call it from group details

### PR DESCRIPTION
**Pull request by matoval**
Jira: https://issues.redhat.com/browse/AAP-19626

Add run command wizard and call it from group details

### CODE REVIEW

File: frontend/awx/resources/inventories/components/RunCommandSteps.tsx [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    { label: t('0 (Normal)'), value: '0' },
+    { label: t('1 (Verbose)'), value: '1' },
+    { label: t('2 (More Verbose)'), value: '2' },
+    { label: t('3 (Debug)'), value: '3' },
+    { label: t('4 (Connection Debug)'), value: '4' },
+    { label: t('5 (WinRM Debug)'), value: '5' },
+  ];
+
+  return (
+    <PageFormSection singleColumn>
```
Comment (MilanPospisil) [inline frontend/awx/resources/inventories/components/RunCommandSteps.tsx:53]: Missing labelHelps for the form. Please include them for all fields and also for other forms in wizard.
image

File: frontend/awx/resources/inventories/components/RunCommandSteps.tsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    <PageFormSection>
+      <PageFormCredentialSelect
+        name="credentials"
+        label={t('Credentials')}
+        placeholder={t('Add credentials')}
+        labelHelpTitle={t('Credentials')}
+        labelHelp={t(
+          'Select credentials for accessing the nodes this job will be ran against. You can only select one credential of each type. For machine credentials (SSH), checking "Prompt on launch" without selecting credentials will require you to select a machine credential at run time. If you select credentials and check "Prompt on launch", the selected credential(s) become the defaults that can be updated at run time.'
+        )}
+        isMultiple
```
Comment (MilanPospisil) [inline frontend/awx/resources/inventories/components/RunCommandSteps.tsx:96]: In old app, its single selection. Is that intentional?

File: frontend/awx/resources/inventories/components/RunCommandSteps.tsx [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+        name="module"
+        placeholderText={t('Select a module')}
+        isRequired
+        label={t('Module')}
+        options={moduleOptions}
+      />
+      <PageFormTextInput
+        name="module_args"
+        placeholder={t('Enter arguments')}
+        isRequired
```
Comment (MilanPospisil) [inline frontend/awx/resources/inventories/components/RunCommandSteps.tsx:64]: Arguments are not required in the old app.

File: cypress/e2e/awx/resources/inventoryGroup.cy.ts [hunk 4]
```diff
@@ -86,15 +86,90 @@ describe('Inventory Groups', () => {
...
-      //5) Navigate back to the Inventory -> Jobs Tab to assert that the Run Command job shows up there
-      //});
+    it('can run an ad-hoc command against a group', () => {
+      cy.createInventoryHostGroup(organization).then((result) => {
+        const { inventory, group } = result;
+        //1) Use the inventory created in beforeEach block, access the groups tab of that inventory
+        cy.navigateTo('awx', 'inventories');
+        cy.filterTableBySingleSelect('name', inventory.name);
+        cy.clickTableRowLink('name', inventory.name, { disableFilter: true });
+        cy.verifyPageTitle(inventory.name);
```
Comment (RedOrNot) [inline cypress/e2e/awx/resources/inventoryGroup.cy.ts:96]: will you be able to reduce code duplication with navigation function?
from line 90 to 99 ?
there are a lot of instances of this lines with minor changes in them

Comment (matoval) [reply cypress/e2e/awx/resources/inventoryGroup.cy.ts:96]: Do you have an example of this?

**Review by resoluteCoder — APPROVED** [review]

File: cypress/e2e/awx/resources/inventoryGroup.cy.ts (hunk 4 above)
Comment (matoval) [reply cypress/e2e/awx/resources/inventoryGroup.cy.ts:96]: This can be addressed in a future ticket

**Pull request merged**